docs/sources/flow/getting-started/migrating-from-static.md

```diff
@@ -305,7 +331,7 @@ After the configuration is converted, review the {{< param "PRODUCT_NAME" >}} co
 
 Review the following checklist:
 
-* The following configuration options are not available for conversion to {{< param "PRODUCT_NAME" >}}: [Integrations next][], [Traces][], and [Agent Management][].
+* The following configuration options are not available for conversion to {{< param "PRODUCT_NAME" >}}: [Traces][] and [Agent Management][].
```

Comment:
Maybe a bit of a scope creep, but should we clarify that traces themselves are supported? It was one of the recent questions we got.

Reply:
added some wording, LMK what you think